core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -219,7 +227,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
         graph.setNode(clusterId, {style: 'fill: ' + color, labelStyle: 'fill: #fff'});
 
         // this is a non-standard use of ReactDOMServer, but it's the cleanest way to unify DagreD3 with React
-        const html = ReactDOMServer.renderToString(<StageStatistics key={stage.id} stage={stage}/>);
+        const html = ReactDOMServer.renderToString(<StageStatistics key={stage.id} stage={stage} query={this.state.query}/>);
```

Comment:
What is this change doing ? Could you add a screenshot of this change ?

Reply:
> What is this change doing ? Could you add a screenshot of this change ?

It was because we need to inject the query stats into the `Live Plan` UI for total CPU time, total scheduling time, etc.  
It looks like below(I will remove the blocking time percentage):  
![image](https://github.com/trinodb/trino/assets/26461591/a8128b0a-f4c7-426e-b57a-c80763605db0)